tests/test_merge.py

```diff
@@ -672,7 +672,7 @@ def test_into_custom_resolver_that_throws(restore_resolvers: Any) -> None:
     def fail() -> None:
         raise ValueError()
 
-    OmegaConf.register_resolver("fail", fail)
+    OmegaConf.register_new_resolver("fail", fail)
```

Comment:
ditto, test that both and new resolvers.

Reply:
Done in c886ad4